spec/forms/project_form_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+describe ProjectForm do
+  # describe "validations" do
+
+    # before do
+      # allow_any_instance_of(SlackAdapter).to receive(:find_channel_by_name).and_return(false)
```

Comment:
Line is too long. [95/80]<br>Incorrect indentation detected (column 6 instead of 4).